docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScale.cs

```diff
@@ -42,18 +43,18 @@ public static void Example()
             var transformedData = pipeline.Fit(data).Transform(data);
 
             // The transformedData IDataView contains the loaded images column, and the grayscaled column.
-            // Preview of the transformedData
-            var transformedDataPreview = transformedData.Preview();
-
-            // Preview of the content of the images.tsv file
-            // The actual images, in the Grayscale column are of type System.Drawing.Bitmap.
-            //
-            // ImagePath    Name        ImageObject                   Grayscale
-            // tomato.bmp   tomato      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-            // banana.jpg   banana      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-            // hotdog.jpg   hotdog      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-            // tomato.jpg   tomato      {System.Drawing.Bitmap}     {System.Drawing.Bitmap}
-
+            
+            // Preview 1 row of the transformedData. 
+            var transformedDataPreview = transformedData.Preview(1);
+            foreach (var kvPair in transformedDataPreview.RowView[0].Values)
+            {
+                Console.WriteLine("{0} : {1}", kvPair.Key, kvPair.Value);
+            }
+            
+            // ImagePath : tomato.bmp
+            // Name : tomato
+            // ImageObject : System.Drawing.Bitmap
+            // Grayscale : System.Drawing.Bitmap
```

Comment:
maybe transpose, to give the column/row idea more accurately.

Reply:
fixed

---
In reply to: [271135652](https://github.com/dotnet/machinelearning/pull/3165#discussion_r271135652) [](ancestors = 271135652)